zcutil/fetch-params.sh

```diff
@@ -2,7 +2,11 @@
 
 set -eu
 
-PARAMS_DIR="$HOME/.zcash-params"
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    PARAMS_DIR="$HOME/Library/Application Support/ZcashParams"
+else
+    PARAMS_DIR="$HOME/.zcash-params"
+fi
```

Comment:
Why is it necessary to change the directory where these params are installed?

Reply:
The changed directory is in fact where `zcashd` has always [looked for the parameters on MacOS](https://github.com/zcash/zcash/blob/master/src/util.cpp#L478-L507); this script was just never downloading to the correct location, which wasn't noticed because we weren't testing it on Macs at the time.

Reply:
Great thanks

Reply:
This seems to be the params directory for macOS
https://github.com/zcash/zcash/blob/d88a12262cc18177a6920ce950507f722edf8ab9/src/util.cpp#L501

Reply:
With macOS, `~/Library/Application Support/...` is where applications should by *convention* write their data.  Btw, since macOS is still Unix under the hood, it could still be be written to ~/.zcash-params and work fine, but it would not be user friendly to place it there.